### DISCUSSION TITLE
Sequel Ace 4.0.7 not supported on this Mac (M1 Apple Silicon, OS 13.5)

### DISCUSSION THREAD

**Discussion by scottmcculloch** [Sequel-Ace/Sequel-Ace#1820] [Q&A]
Hi,

I've been using Sequel Ace on this Mac for some time, but don't use it regularly (last time was probably a month ago)... I tried to launch it today and got an error message telling me that it's not supported on this Mac.

Is this a known issue? Is there anything I can do to get it working?

Thanks,
Scott

**Comment by samuelbradshaw** [comment]
I'm on an M1 Apple Silicon Mac with macOS 13.5, and Sequel Ace 4.0.7 is working for me. Did you install it from the Mac App Store, or somewhere else?

**Reply by scottmcculloch** [reply]
Mac App Store. I should also note, in Finder it's showing as faded with a circle and line through it.
Maybe I'll try deleting and then redownloading (hoping I don't lose all of my saved connections).

**Reply by scottmcculloch** [reply]
Thanks @samuelbradshaw -- since it's working for you, I decided to go ahead and delete it and re-dowload it from the Mac App Store... I'm back in business!

**Reply by samuelbradshaw** [reply]
Nice! Maybe the Mac App Store lost its connection in the middle of updating the app, or something like that. Glad it's working!
